src/pages-conditional/eth.tsx

```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
-`
+const Content = (props: ChildOnlyProp) => (
+  <Box w="full" px={8} py={4} {...props} />
+)
 
-const StyledTwoColumnContent = styled(TwoColumnContent)`
-  align-items: flex-start;
-`
+const Divider = () => (
+  <Box my={16} mx={0} w="10%" h={1} backgroundColor="homeDivider" />
+)
 
-const Subtitle = styled.p`
-  font-size: 1.25rem;
-  line-height: 140%;
-  color: ${(props) => props.theme.colors.text200};
-`
-const SubtitleTwo = styled.p`
-  font-size: 1.25rem;
-  line-height: 140%;
-  margin-bottom: 2rem;
-  color: ${(props) => props.theme.colors.text300};
-`
+const GrayContainer = (props: ChildOnlyProp) => (
+  <Box
+    width="full"
+    py={16}
+    mt={8}
+    background="grayBackground"
+    boxShadow="inset 0px 1px 0px var(--eth-colors-tableItemBoxShadow)"
+    {...props}
+  />
+)
 
-const StyledEthPriceCard = styled(EthPriceCard)`
-  margin-bottom: 2rem;
-`
+const Intro = (props: ChildOnlyProp) => (
+  <Box maxW="608px" mb={{ base: 12, sm: 16 }} {...props} />
+)
 
-const HeroContainer = styled.div`
-  display: flex;
-  justify-content: space-between;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    flex-direction: column-reverse;
-  }
-`
+const StyledCard = (props: ComponentProps<typeof Card>) => (
+  <Card
+    m={4}
+    p={6}
+    flex="1 0 30%"
+    minW="280px"
+    maxW={{ base: "full", md: "46%", lg: "31%" }}
+    {...props}
+  />
+)
 
-const Hero = styled(GatsbyImage)`
-  flex: 1 1 100%;
-  max-width: 800px;
-  align-self: center;
-  background-size: cover;
-  background-repeat: no-repeat;
-  margin-left: 2rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    margin-left: 0;
-    width: 100%;
-  }
-`
+const Slogan = (props: ChildOnlyProp) => (
+  <Text
+    textStyle="normal"
+    fontWeight="normal"
+    fontSize="2rem"
+    lineHeight={1.4}
+    {...props}
+  />
+)
 
-const Header = styled.header`
-  flex: 1 1 50%;
-  min-width: 300px;
-  margin-top: 8rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
-    margin-top: 1.5rem;
-  }
-`
+const Title = (props: ChildOnlyProp) => (
+  <Heading
+    as="h1"
+    fontSize="sm"
+    lineHeight={1.4}
+    letterSpacing="0.04em"
+    fontWeight="500"
+    mb={4}
+    mt={0}
+    textTransform="uppercase"
+    color="textTableOfContents"
+    {...props}
+  />
+)
 
-const H2 = styled.h2`
-  margin-top: 0;
-`
+const Subtitle = (props: ChildOnlyProp) => (
+  <Text fontSize="xl" lineHeight={1.4} color="text200" {...props} />
+)
 
-const H4 = styled.h4`
-  font-weight: 600;
-`
+const TwoColumnContent = (props: FlexProps) => (
+  <Flex
+    w="full"
+    direction={{ base: "column", lg: "row" }}
+    justify="space-between"
+    p={8}
+    mb={12}
+    {...props}
+  />
+)
 
-const StyledCardContainer = styled(CardContainer)`
-  margin-bottom: 2rem;
-`
+export const LeftColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="0 0 50%"
+    maxW={{ base: "full", lg: "75%" }}
+    mr={{ lg: 16 }}
```

Comment:
`me` for `margin-inline-end` (rtl responsive)
```suggestion
    me={{ lg: 16 }}
```